myblog/blog/api/serializers.py

```diff
@@ -7,7 +7,7 @@ class BlogPostSerializer(serializers.ModelSerializer):
     
     class Meta:
         model = BlogPost
-        fields = ('body', 'title', 'author')
+        fields = '__all__'
```

Comment:
Usually is not such a good idea to use these kinds of wildcards because if you update something on the model you can deliver data to the user that maybe you don't intend to give (for performance or security reasons), is better to explicitly declare which fields will be visible.